data_model/1.3/clusters/ColorControl.xml

```diff
@@ -74,32 +74,32 @@ Davis, CA 95616, USA
     <feature bit="0" code="HS" name="Hue/Saturation" summary="Supports color specification via hue/saturation.">
       <optionalConform/>
     </feature>
-    <feature bit="1" code="EHUE" name="Enhanced Hue" summary="Enhanced hue is supported.">
+    <feature bit="1" code="EHUE" name="Enhanced" summary="Enhanced hue is supported.">
       <optionalConform/>
     </feature>
-    <feature bit="2" code="CL" name="Color loop" summary="Color loop is supported.">
+    <feature bit="2" code="CL" name="Color" summary="Color loop is supported.">
```

Comment:
This is properly not going to cause to much issue and will be fixed given the editorial for ColorControl have finally landed, but the feature names are getting cut off and CL/CT has the same name in this scrape

Reply:
see description above - opened a DM editor issue here - https://github.com/csa-data-model/projects/issues/502

To be fair, I don't think those feature names are actually valid - I'm not sure how we correlate feature names in the device library, but I suspect we're asking for trouble with these spaces.